### DISCUSSION TITLE
v1.4.4

### DISCUSSION THREAD

**Discussion by sgkoishi** [sgkoishi/PackTracker#3] [General]
New feature: show history packs ever granted (credit to Firestone https://www.firestoneapp.com)

<hr /><em>This discussion was created from the release <a href='https://github.com/sgkoishi/PackTracker/releases/tag/v1.4.4'>v1.4.4</a>.</em>

**Comment by TheMrKochan** [comment]
Hi, could you attach some screenshots with a description of the new feature?

**Reply by sgkoishi** [reply]
[Here it is!](https://github.com/sgkoishi/PackTracker/discussions/3#discussioncomment-601605)

**Comment by sgkoishi** [comment]
It can show how many packs you've ever got in total, even if HDT or this plugin is not installed.
<hr />

![image](https://user-images.githubusercontent.com/9637711/114446519-2c888700-9b86-11eb-9f9a-fc6130dae3dc.png)
Here's the result of 17 tracked packs and 15 untracked.

**Reply by TheMrKochan** [reply]
Big thx, can you add this to readme file on main page?

**Reply by sgkoishi** [reply]
> 
> 
> I double-checked, but this feature does not work for me... It should show absolutely all packages, even those when the plugin did not work?
> ![image](https://user-images.githubusercontent.com/75619070/114449924-08589480-9bde-11eb-8de2-7152af5c66c2.png)

Those data are read from the game directly so it requires your game launched.
Three possibilities:
* `123 Packs (Tracked)`: Game launched, all packs tracked
* `123 Tracked / 456 Total`: Game launched, some packs tracked
* `123 Packs`: Game not launched, the plugin have no idea how many packs you've ever got

**Reply by TheMrKochan** [reply]
> > I double-checked, but this feature does not work for me... It should show absolutely all packages, even those when the plugin did not work?
> > ![image](https://user-images.githubusercontent.com/75619070/114449924-08589480-9bde-11eb-8de2-7152af5c66c2.png)
> 
> Those data are read from the game directly so it requires your game launched.
> Three possibilities:
> 
> * `123 Packs (Tracked)`: Game launched, all packs tracked
> * `123 Tracked / 456 Total`: Game launched, some packs tracked
> * `123 Packs`: Game not launched, the plugin have no idea how many packs you've ever got

Yes, I already understood) I'm sorry, I'm a little dumb at this!

**Reply by TheMrKochan** [reply]
![image](https://user-images.githubusercontent.com/75619070/114452919-abf77400-9be1-11eb-9036-3d6a25740aaa.png)
OK, I decided to translate the extension into Russian, but something went wrong and now the percentages and any text in general are not displayed

**Reply by sgkoishi** [reply]
> 
> 
> ![image](https://user-images.githubusercontent.com/75619070/114452919-abf77400-9be1-11eb-9036-3d6a25740aaa.png)
> OK, I decided to translate the extension into Russian, but something went wrong and now the percentages and any text in general are not displayed

I guess you change some binding? The longest streak is still working and you may try revert some of you change one by one

**Reply by TheMrKochan** [reply]
> I guess you change some binding? The longest streak is still working and you may try revert some of you change one by one

yes, the problem was in the PackTracker.csproj file
